common/src/main/java/com/wynntils/services/chat/ChatTabService.java

```diff
@@ -221,4 +221,26 @@ private boolean matchMessageFromEvent(ChatTab chatTab, ClientsideMessageEvent ev
 
         return event.getOriginalStyledText().matches(regex.get());
     }
+
+    /**
+     * Sends a chat message respecting chat tab autocommand settings.
+     * If no chat tab is actively selected, the message will be sent normally.
+     * @param message The message to send.
+     */
+    public void sendChat(String message) {
```

Comment:
So this is like "send message but prefix with auto command if present"? Hm...

Reply:
Yes, it sends with prefix if there is one. But I think the name is fine, it has to be called via Services.ChatTab.sendChat() anyway, so it is pretty clear it is related to the chat tab prefix stuff.